data_architect_misc/data_transformer/constants/comp_harm_constants.py

```diff
@@ -209,15 +214,15 @@
     "(?i).*Moisturizer.*": "Personal Care",
     "(?i).*MOUTH.*CARE.*": "Personal Care",
     "(?i)^PC$": "Personal Care",
-    "(?i).*PERSONAL.*CARE.*": "Personal Care",
+    "(?i).*[A-z]PERSONAL.*[A-z]CARE.*": "Personal Care",
```

Comment:
Why do we need this [A-z]? Can you give me an example of a word that fits this REGEX?